.github/workflows/python-package-conda.yml

```
@@ -1,5 +1,13 @@
-name: Python Package using Conda
-
+name: myenv
+channels:
```

actionlint (1.7.4)

unexpected key "channels" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)